lib/core/spec/lib/jwt_spec.rb

```diff
@@ -3,5 +3,60 @@
 require 'rails_helper'
 
 RSpec.describe Ros::Jwt do
-  pending 'something'
+  subject { described_class.new('Bearer something') }
+
+  context 'when payload is a hash' do
```

Comment:
You should make sure that the payload passed is maintained and if it includes anything that should go back to defaults, then it gets overridden.

E.g
```
subject { described_class.new({ aud: 'bananas', something: 'else' }) }
```